### ISSUE TITLE
Training Time

### ISSUE THREAD

**Issue by leviethung2103** [HyeongseokSon1/SRFeat#6]
Hello,

I would like to ask you about the training time for pretrained Generator with MSE Loss. How long does it take to finish your training in pretrained model and in full model ? 

Furthermore, how many GPUs you have used ?

Thank you,

**Comment by HyeongseokSon1** [comment]
We made around 100,000 patches (LR:74*74, HR:296*296 size) from DIV2K
and pretrained the generator network for 20 epochs.
And we trained the full model for 4 epochs.
It took about 2 days and 1 day separately by a single Titan Xp GPU.
